edi_voxel_account_invoice_oca/data/queue_job_function_data.xml

```diff
@@ -0,0 +1,14 @@
+<!-- Copyright 2023 Tecnativa - Ernesto tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record id="job_function_get_and_send_voxel_report" model="queue.job.function">
```

Comment:
Shouldn't this requires a migration script to find the existing functions and channels and put them the XML-IDs?

Reply:
If not, I think there will be unique constraint errors.